### ISSUE TITLE
calling apply or call on NativeFunction with no parameters fails.

### ISSUE THREAD

**Issue by taviso** [frida/frida#1074]
Calling `.apply`  or `call` on a `NativeFunction` object works, except in the case of no parameters.

This seems like it should work.

```javascript
// This will work
var f = new NativeFunction(ptr(0x123), "void", ["int"])
f.apply(f, [1])
// This will work
var f = new NativeFunction(ptr(0x123), "void", ["int", "int"])
f.apply(f, [1, 2])
// This will fail, Error: bad argument count
var f = new NativeFunction(NULL, "void", [])
f.apply(f, [])
```

**Comment by eugenekolo** [comment]
Use ['void']

**Comment by eugenekolo** [comment]
Actually it worked fine for me either way

```[Local::PID::1124]-> var f = new NativeFunction(ptr(0x123), "void", [])                                                                                    
undefined
[Local::PID::1124]-> f.apply(f, [])                                                                                                                        
Error: access violation accessing 0x123
```

**Comment by taviso** [comment]
Hmm, not sure what I'm doing wrong, I'm testing it by typing that (exactly what you've written) into the debug console.

Perhaps this is already fixed in master or something?

**Comment by oleavr** [comment]
Thanks, nice catch! This seems like a bug in the V8 runtime. I can reproduce it with `frida -p 0 --runtime=v8` but not `frida -p 0 --runtime=duk`. Will investigate.

**Comment by oleavr** [comment]
Just released 12.7.19 with the fix. Thanks for your help guys!
